index.html

```diff
@@ -45,7 +46,7 @@ <h1>Login</h1>
            
             <input class="checkbox" type="checkbox">Remember me</input>
```

Comment:
Increase Remember me checkbox size for better visibility

Reply:
Thank u